src/i18n/en/ui.ts

```diff
@@ -60,6 +60,9 @@ export default {
 	'fallbackContent.notice':
 		'This page is not yet available in your language, so we’re showing you the English version. You can help by translating it!',
 	'fallbackContent.linkText': 'Learn more about how you can contribute',
+	// Outdated content notice shown when a page is outdated
+	'outdatedContent.notice':
+		'The content on this page is an older translation that requires updating to match the current English version. You can help by translating it! Visit our [contributing](/en/contribute/) page for details.',
```

Comment:
```suggestion
		'The content on this page is an older translation that requires updating to match the current English version. You can help by translating it!',
```
Checked the Netlify preview and it didn't like the Markdown syntax for a link. Plus, we already have a link directly to the translating page that is in the page's own language, so I think that's good enough!